contracts/conwaygol.scrypt

```diff
@@ -2,7 +2,7 @@ import "util.scrypt";
 
 // Conway Game Of Life on a board of N * N
 contract GameOfLife {
-    static int N = 10;
+    static int N = 7;
```

Comment:
Changed to a 7x7 to make tests easier.

Worth noting I could not deploy anything larger than 4x4 to mainnet without having a miner with custom settings accept it.